spec/rubocop/cop/style/conditional_assignment_assign_to_condition_spec.rb

```diff
@@ -1192,6 +1192,26 @@
       RUBY
     end
 
+    it 'corrects assignment to unbracketed array in if else' do
+      source = <<-RUBY.strip_indent
+        if foo
+          bar = 1
+        else
+          bar = 2, 5, 6
+        end
+      RUBY
+
+      new_source = autocorrect_source(source)
+
+      expect(new_source).to eq(<<-RUBY.strip_indent)
+        bar = if foo
+          1
+        else
+          [2, 5, 6]
+              end
```

Comment:
I guess this happens because of interference from `Lint/EndAlignment`?

Reply:
Actually, `Style/ConditionalAssignment` realigns `end` keyword by itself. Please see #2719 / #2800 for details.

Reply:
That looks extremely strange.

Reply:
@bbatsov 

Agreed. Would it be better to have an "explanatory" variable? i.e.

```ruby
indent = ' ' * 'bar = '.length
expect(new_source).to eq(<<-RUBY.strip_indent)
  bar = if foo
    1
  else
    [2, 5, 6]
  #{indent}end
RUBY
```

If so, I'd correct [the other case](https://github.com/akhramov/rubocop/blob/8cc74135877bb619a839ae863ca82f15d0567520/spec/rubocop/cop/style/conditional_assignment_assign_to_condition_spec.rb#L1186-L1192), from which I've derived the style.

Reply:
Is it possible to configure the auto-correct to produce a more ... reasonable style in the tests? 

Either:

```ruby
bar = if foo
        1
      else
        [2, 3]
      end
```

or:

```ruby
bar = if foo
  1
else
  [2, 3]
end
```

Reply:
@Drenmi, thanks for guidance! Fixed.

Reply:
Well done, @akhramov! Looks much better now. 🙂